python/tritonserver/_api/_model.py

```diff
@@ -180,33 +177,12 @@ def async_infer(
         if inference_request is None:
             inference_request = InferenceRequest(model=self, **kwargs)
 
-        if (inference_request.response_queue is not None) and (
-            not isinstance(inference_request.response_queue, asyncio.Queue)
-        ):
-            raise InvalidArgumentError(
-                "asyncio.Queue must be used for async response iterator"
-            )
-
+        # The 'request' is referencing pointers managed by the 'inference_request', so
```

Comment:
this note seems unnecessary here - would we add it instead to the asyncresponseiterator object? That's where we maintain that relationship is that correct?

Reply:
sure, updated.